e2e/tests/regression/test_basic.robot

```diff
@@ -0,0 +1,95 @@
+*** Settings ***
+Documentation    Basic Test Cases
+
+Test Tags    regression
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/recurringjob.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/snapshot.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Volume Basic Test
+    [Tags]  coretest
+    [Documentation]    Test basic volume operations
+    ...
+    ...                1. Check volume name and parameter
+    ...                2. Create a volume and attach to the current node, then check volume states
+    ...                3. Check soft anti-affinity rule
+    ...                4. Write then read back to check volume data
+    Given Create volume wrong_volume-name-1.0
+    Then No volume created
+
+    Given Create volume wrong_volume-name
+    Then No volume created
+
+    Given Create volume 0 with frontend invalid_frontend
+    Then No volume created
+
+    Given Create volume 0 with 2 GB and 3 replicas
+    When Attach volume 0
+    And Wait for volume 0 healthy
+    Then Validate volume 0 replicas anti-affinity
+
+    When Write data to volume 0
+    Then Check volume 0 data is intact
+
+    And Detach volume 0
+    And Delete volume 0
+
+Test Snapshot
```

Comment:
I think we should re-organize the tests to improve on the groupings.
```
- e2e/tests/
   +--- volume
   +--- snapshot
   +--- setting
   +--- etc.
```

Reply:
@c3y1huang Are `volume`, `snapshot` folders? Could you suggest the robot filename?

Reply:
At the moment, I believe organizing them into folders isn't necessary unless we establish clear groupings for each resource test. Files should suffice for now. What do you think?
```
- e2e/tests/
   +--- volume.robot
   +--- snapshot.robot
   +--- setting.robot
   +--- etc.
```

Reply:
Refined. Thank you!